lang/py/.gitignore

```diff
@@ -9,5 +9,6 @@ avro/VERSION.txt
 avro/interop.avsc
 avro/tether/InputProtocol.avpr
 avro/tether/OutputProtocol.avpr
+avro/test/interop/
```

Comment:
Is this too general?

Reply:
nah

Reply:
This isn't part of the release artifact, so I'm fine with it 👍